index.html

```diff
@@ -20,6 +20,9 @@ <h1>Instagram Clone</h1>
     <div id="detailed-view"></div>
     </center>
     <!-- usually call js right before end of body tag -->
+    <script src="https://code.jquery.com/jquery-3.3.1.min.js"
```

Comment:
this looks pretty good 👍 
***great job***

Reply:
Thanks